test/gui/shared/steps/steps.py

```diff
@@ -1298,11 +1298,8 @@ def step(context):
 
 @Then('the sync all checkbox should be checked')
 def step(context):
-    newAccount = AccountConnectionWizard()
-    test.compare(
-        waitForObjectExists(newAccount.SYNC_DIALOG_ROOT_FOLDER).checkState,
-        "checked",
-        "Assert sync all checkbox is checked",
```

Comment:
Do we need `SYNC_DIALOG_ROOT_FOLDER` anywhere else? Or can you assign `names.add_Folder_Sync_Connection_qt_wizard_finish_QPushButton` to `SYNC_DIALOG_ROOT_FOLDER`? Does changing the value of the variable makes sense? Can we reuse the variable names, even though some selectors have changed.

Reply:
I looked into the code. this variable is being used by step `When the user selects the following folders to sync:` in scenario `Scenario: Sync only one folder from the server`. So may be we can not remove this variable for now. How about putting `names.add_Folder_Sync_Connection_qt_wizard_finish_QPushButton` into some variable in the page object? It is only used in one place, so it can be easily migrated to a page object.